manifests/params.pp

```diff
@@ -268,6 +268,7 @@
   $server_startsnmptrapper                  = '0'
   $server_starttimers                       = '1'
   $server_starttrappers                     = '5'
+  $server_startpreprocessors                = undef
```

Comment:
if I read the change correctly, both new variables from params.pp are only used once. In that case please directly assign undef in init.pp / proxy.pp. That makes it later on easier to document the changes with puppet-strings.